### ISSUE TITLE
Date Reset

### ISSUE THREAD

**Issue by philipp-andreas** [budulinek/Daikin-P1P2---UDP-Gateway#5]
Hello,
I use P1P2-udp on a hoval heating Pump... i use it only serial.
all works as expected.
i get the serial string and it is possible to write serial packets like 35400001 to turn dhw on etc.
 
the only problem is, if i connect the arduino on the P1P2 bus, the date stored in the main controller, resets to 2016. (the install year of the Pump). if i set the date to 2021... after a few seconds...2016. day and month are still correct.

if i use P1P2 monitor from arnold, the problem don´t extist.

lg Andreas

**Comment by budulinek** [comment]
Oh, what a coincidence! Just yesterday I have noticed the same thing on my Altherma (2021  going back to 2016). Well, actually my wife noticed that we have a wrong date on Altherma...

So I can confirm the issue. 

I do not know if it is a related issue, but I also had wrong time (which caused problems to my DHW schedule). Looking at my data log, my time and date got reset on 31.10. when Altherma was supposed to change to winter time. I was able to set correct time and date, with the exception of year, as you mentioned.
